src/frontend/screens/Observation/index.tsx

```diff
@@ -96,33 +114,52 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
             />
           </Text>
         </View>
-        <View style={[styles.section, {flex: 1}]}>
-          <PresetHeader preset={preset} />
-
-          {typeof observation.tags.notes === 'string' ? (
-            <View style={{paddingTop: 15}}>
-              <Text style={styles.textNotes}>{observation.tags.notes}</Text>
-            </View>
-          ) : null}
-          {attachments.length > 0 && (
-            <MediaScrollView
-              attachments={attachments}
-              observationId={observationId}
-            />
-          )}
-        </View>
-        {fields.length > 0 && (
-          <FieldDetails observation={observation} fields={fields} />
-        )}
-        <View style={styles.divider} />
-        {isDeviceInfoPending || isDeviceIdPending ? (
-          <UIActivityIndicator size={20} />
+        {track === 'loading' ? (
+          <Loading />
         ) : (
-          <ButtonFields
-            isMine={isMine}
-            observationId={observationId}
-            fields={fields}
-          />
+          <>
+            <View style={[styles.section, {flex: 1}]}>
```

Comment:
I'm curious about the necessity of applying flex: 1 in so many places. Is there a specific reason for this repetition? This View has flex: 1 here and styles.section also has flex: 1. And then below, the root also has flex: 1.